src/lifetracker/calendar/CalendarEntryImpl.java

```diff
@@ -90,11 +88,21 @@ public EntryType getType() {
     }
 
     @Override
+    public void setType(EntryType entryType) {
+        this.entryType = entryType;
```

Comment:
I don't think it's a good idea to let external classes change the entry type. You might end up with a event with null start and end dates.